src/theme/index.js

```diff
@@ -7,11 +8,11 @@ const AvatarWrapper = styled.div`
   border-radius: 50%;
   width: ${props => props.size};
   height: ${props => props.size};
-  border: 1px solid ${props => props.borderColor};
+  border: 1px solid ${props => hexToRgba(props.borderColor, props.borderOpacity)};
   cursor: pointer;
   transition: border ${props => props.animationTime};
   &:hover {
-    border: 1.2px solid #000;
+    border: 1.2px solid ${props => hexToRgba(props.borderColor, 0.1)};
```

Comment:
Great. I think that 0.4 it will be better.

Reply:
ok, done ^^